build/copy_info_plist.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+Copies the Info.plist and adds extra fields to it like the git hash of the
+engine.
+
+Precondition: $CWD/../../flutter is the path to the flutter engine repo.
+
+usage: copy_info_plist.py <src_path> <dest_path>
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import sys
+import git_revision
+import os
+
+def main():
+  text = open(sys.argv[1]).read()
+  engine_path = os.path.join(os.getcwd(), "..", "..", "flutter")
+  revision = git_revision.GetRepositoryVersion(engine_path)
+  text = text.replace("</dict>", """  <key>EngineVersion</key>
```

Comment:
Would it make more sense to actually use an XML parser for this?  If we add more features to this it may get harder to track.

Alternatively, we could just have the whole plist string in here as a template and use something like `String.format`, like in https://github.com/flutter/engine/blob/master/tools/gen_android_buildconfig.py

Reply:
I considered both things but decided against them.  There wasn't a compelling reason to make it more complicated.  I didn't want to fiddle much with the formatting and some xml parsers trip up on schemas.  Ultimately I went with KISS.  I'm open to either of those approaches if it becomes easier / more robust.

Reply:
What's the failure mode if someone someday adds another dictionary entry to Info.plist?

Reply:
Ok, I forgot dictionaries were valid in an Info.plist.  I'll switch it to the template implementation in a sec.

Reply:
Done.